extensions/che-theia-user-preferences/src/node/che-theia-preferences-contribition.ts

```diff
@@ -0,0 +1,31 @@
+/********************************************************************************
+ * Copyright (C) 2018-2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject } from "inversify";
+import { BackendApplicationContribution } from '@theia/core/lib/node/backend-application';
+import { CheTheiaUserPreferencesSynchronizer } from './che-theia-preferences-synchronizer';
+
+@injectable()
+export class CheTheiaPreferencesContrinution implements BackendApplicationContribution {
```

Comment:
hello, FYI name of the file and name of the class have a typo

name of file should contain `contribUtion` and class `contriBution` (n/b)

Reply:
Yes, thanks you! Will fix